diff --git a/intel-ipsec-mb/lib/Makefile b/intel-ipsec-mb/lib/Makefile
index ddb4cb4..61dadbf 100644
--- a/intel-ipsec-mb/lib/Makefile
+++ b/intel-ipsec-mb/lib/Makefile
@@ -83,8 +83,9 @@ CC ?= gcc
 
 # if "-z ibt" is supported then assume "-z shstk, -z cet-report=error" are also supported
 # "-fcf-protection" needs to be checked separately
-CC_HAS_CET = $(and $(shell $(CC) --target-help 2> /dev/null | grep -m1 -e "-z ibt" | wc -l), \
-	$(shell $(CC) --help=common 2> /dev/null | grep -m1 -e "-fcf-protection" | wc -l))
+#CC_HAS_CET = $(and $(shell $(CC) --target-help 2> /dev/null | grep -m1 -e "-z ibt" | wc -l), \
+#	$(shell $(CC) --help=common 2> /dev/null | grep -m1 -e "-fcf-protection" | wc -l))
+CC_HAS_CET = 0
 CET_LDFLAGS=-r -z ibt -z shstk
 
 CFLAGS := -DLINUX -DNO_COMPAT_IMB_API_053 $(EXTRA_CFLAGS) $(INCLUDES) \
